dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -89,30 +102,30 @@ ImmutableList<Task<?>> createTasks(ConnectorArguments arguments, Duration querie
       builder.add(StatsJdbcTask.fromQuery(query));
     }
     for (String name : NATIVE_NAMES_OPTIONAL) {
-      builder.addAll(createTaskWithAlternative(name, /* isRequired= */ false, queriedDuration));
+      QueryGroup group = QueryGroup.create(/* isRequired= */ false, NATIVE, SINGLE_TENANT);
```

Comment:
I'm looking at the `QueryGroup` class and the first parameter to `create` method is `required`, not `isRequired`.

Reply:
Ah, true. The literals' comments keep popping up, plain VS Code isn't much help with them. I fixed it to say `required` consistently